documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,256 @@
+
+Before starting the migration, verify that your environment can support Kafka in KRaft mode, as there are a number of xref:ref-operator-use-kraft-feature-gate-str[limitations].
+Note also, the following:
+
+* Migration is only supported on dedicated controller nodes, not on nodes with dual roles as brokers and controllers.
+* Throughout the migration process, ZooKeeper and Controller nodes operate in parallel for a period, requiring sufficient compute resources in the cluster.
+
+.Prerequisites
+
+* You must be using Strimzi 0.40 or newer with Kafka 3.7.0 or newer. If you using an earlier version of Strimzi or Apache Kafka, upgrade before migrating to KRaft mode.
+* Verify that the ZooKeeper-based deployment is operating without the following, as they are not supported in KRaft mode:
+** The Topic Operator running in bidirectional mode. It should either be in unidirectional mode or disabled.
+** JBOD storage. While the `jbod` storage type can be used, the JBOD array must contain only one disk.
+* The Cluster Operator that manages the Kafka cluster is running.
+* The Kafka cluster deployment uses Kafka node pools.
++
+If your ZooKeeper-based cluster is already using node pools, it is ready to migrate.
+If not, you can xref:proc-migrating-clusters-node-pools-str[migrate the cluster to use node pools]. 
+To migrate, brokers must be contained in a `KafkaNodePool` resource configuration that is assigned a `broker` role and has the name `kafka`.
+Support for node pools is enabled in the `Kafka` resource configuration using the `strimzi.io/node-pools: enabled` annotation.
+
+In this procedure, the Kafka cluster name is `my-cluster`, which is located in `my-project`. 
+The name of the controller node pool is `controller`.
+
+.Procedure
+
+. For the Kafka cluster, create a node pool with a `controller` role.
++
+Add a quorum of controller nodes to the node pool.
++
+.Example configuration for a controller node pool
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaNodePoolApiVersion}
+kind: KafkaNodePool
+metadata:
+  name: controller
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  replicas: 3
+  roles:
+    - controller
+  storage:
+    type: jbod
+    volumes:
+      - id: 0
+        type: persistent-claim
+        size: 20Gi
+        deleteClaim: false
+    resources:
+      requests:
+        memory: 64Gi
+        cpu: "8"
+      limits:
+        memory: 64Gi
+        cpu: "12"    
+----
++
+NOTE: For the migration, you cannot use a node pool of nodes that share the broker and controller roles.
+
+. Apply the new `KafkaNodePool` resource to create the controllers.
++
+Errors related to using controllers in a ZooKeeper-based environment are expected.
+
+. Enable KRaft migration in the `Kafka` resource by setting the `strimzi.io/kraft` annotation to `migration`:
++
+[source,shell]
+----
+kubectl annotate kafka my-cluster strimzi.io/kraft: migration
```

Comment:
Is this really correct? Shouldn't be this:
```suggestion
kubectl annotate kafka my-cluster strimzi.io/kraft=migration
```

Reply:
Either okay?  We use `:` throughout the docs for these annotate commands

Reply:
With `:` the command fails. I would also add quotation marks, as annotation values must be strings.

```sh
kubectl annotate kafka my-cluster strimzi.io/kraft="migration"
```

Reply:
I have never tried wit `:` and @fvaleri is right, the command doesn't work on my side either.
Only thing the quotes are not needed around the annotation value but anyway the command works with them as well.

Reply:
Thanks. Actually, we only used `:` in a  couple of places (which I must have referenced). I've gone through the annotations and made consistent. @see-quick -- if you could please check commit 
[docs(review): annotations values update](https://github.com/strimzi/strimzi-kafka-operator/pull/9633/commits/1f44abcd630c1eff0b2e2632410317255243f3e5)

Reply:
Update: changes outside this migration will be in another PR